spinta/ufuncs/linkbuilder/ufuncs.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from spinta.core.ufuncs import ufunc, Bind
+from spinta.datasets.backends.wsdl.components import WsdlBackend
+from spinta.exceptions import InvalidSource, SoapServiceError, InvalidValue
+from spinta.ufuncs.linkbuilder.components import LinkBuilder
+
+log = logging.getLogger(__name__)
+
+
+@ufunc.resolver(LinkBuilder, Bind)
+def wsdl(env: LinkBuilder, parent_resource_bind: Bind) -> None:
+    resource_name = parent_resource_bind.name
+
+    if resource_name == env.resource.name:
+        raise InvalidValue(
+            message=f"wsdl() argument {resource_name} must be name of another wsdl type resource."
+        )
+
+    wsdl_resources = (
+        res_name
+        for res_name, res in env.dataset.resources.items()
+        if res.backend.type == "wsdl"
+    )
+    if resource_name not in wsdl_resources:
+        raise InvalidValue(
+            message=f"wsdl() argument {resource_name} must be wsdl type resource."
+        )
+
+    return env.call("wsdl", env.dataset.resources[resource_name].backend)
+
+
+@ufunc.resolver(LinkBuilder, WsdlBackend)
+def wsdl(env: LinkBuilder, parent_resource_backend: WsdlBackend) -> None:
+    soap_source = env.resource.backend.config.get("dsn")
+
+    with parent_resource_backend.begin():
+        if not (client := parent_resource_backend.client):
+            raise SoapServiceError(
+                f'Cannot parse SOAP operation "{soap_source}", because WSDL client is not initialized.'
```

Comment:
Not sure about this, but would it be a good idea to also point out to the user how that is done here? Might be more complex than I think though, so do as you please after reading this